### PR TITLE
issue #2: added docstrings

### LINKED ISSUE #2: Add Docstrings

### PR DESCRIPTION
**Pull request by nityanandagohain**
fixes #2 

Please review. Please tell me if I need to add something more thanks.

### CODE REVIEW

File: git_stalk/stalk.py [hunk 1]
```diff
@@ -32,6 +33,7 @@ def get_event(string):
 
 
 def get_details(event):
+    """Returns the details of the event accordint to the type of the event"""
```
Comment (aashutoshrathi) [inline git_stalk/stalk.py]: According*
Check the spellings.

**Comment by nityanandagohain** [comment]
I have made the change.

File: git_stalk/stalk.py [hunk 2]
```diff
@@ -208,3 +234,4 @@ def run():
 
 if __name__ == '__main__':
     run()
+
```
Comment (aashutoshrathi) [inline git_stalk/stalk.py]: newline

Comment (nityanandagohain) [reply git_stalk/stalk.py]: Should I add one more or remove it?

Comment (aashutoshrathi) [reply git_stalk/stalk.py]: add one

Comment (nityanandagohain) [reply git_stalk/stalk.py]: done @aashutoshrathi

**Pull request merged**